ibexOverview.js

```diff
@@ -0,0 +1,140 @@
+var PORT = 60000;
+var HOST = "http://dataweb.isis.rl.ac.uk" 
+//$('.btn').button();
+
+function refresh() {
+	$.ajax({
+		url: HOST + ":" + PORT + "/",
+		dataType: 'jsonp',
+		data: {"Instrument": "all"},
+		timeout: 1000,
+		error: function(xhr, status, error){ 
+			window.alert("Error: JSON not read");
+		},
+		success: function(data){ 
+			//window.alert(JSON.stringify(data));
+			display_data(data);
+		}
+	});
+}
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function sortObject(o) {
+    var sorted = {},
+    key, a = [];
```

Comment:
Syntactically correct but I think it's better practice to put `var` on each line to highlight this is a continued variable definition.

In fact, `key` should not be defined here, as below.